configure.ac

```
@@ -9,6 +9,10 @@ AC_INIT([Squid Web Proxy],[7.0.0-VCS],[https://bugs.squid-cache.org/],[squid])
 AC_PREREQ(2.61)
 AC_CONFIG_HEADERS([include/autoconf.h])
 AC_CONFIG_AUX_DIR(cfgaux)
+
+# TODO: Require Autoconf v2.70 and upgrade to AC_CONFIG_MACRO_DIRS
+AC_CONFIG_MACRO_DIR([m4])
+
```

Upgrading the minimum required autoconf version is a ongoing cycle. No need to add a TODO about it.
```suggestion
AC_CONFIG_MACRO_DIR([m4])
```

---

> Upgrading the minimum required autoconf version is a ongoing cycle. No need to add a TODO about it.

Agreed, but this TODO is not really about that ongoing cycle. This TODO is useful for two reasons:

* It documents that we should be using AC_CONFIG_MACRO_DIRS() instead of AC_CONFIG_MACRO_DIR() here.
* It documents the minimum version we would have to require to switch to AC_CONFIG_MACRO_DIRS().

Both bits of information was not trivial to find (for me). 

If you are sure that we are better without this TODO, I will remove it without further arguments.

---

The source code comment in question is now gone because we no longer need to add that problematic AC_CONFIG_MACRO_DIR() call at all.